### PR TITLE
KFSPTS-6749: update to 2016-11-03.1 version of Finacials base code

### PR DESCRIPTION
**Pull request by bhutchinson**
This is one of three related PRs, one in config, one in nonprod-sql and this one.

### CODE REVIEW

File: src/test/java/edu/cornell/kfs/fp/batch/service/impl/AdvanceDepositServiceImplTest.java [hunk 1]
```diff
@@ -641,6 +641,11 @@ public PersistableBusinessObject retrieve(PersistableBusinessObject object) {
         }
 
         @Override
+        public <T extends BusinessObject> Collection<T> findMatching(Class<T> aClass, Map<String, ?> map, int i, int i1, String[] strings) {
```
Comment (jhulslander) [inline src/test/java/edu/cornell/kfs/fp/batch/service/impl/AdvanceDepositServiceImplTest.java]: this seems odd, do we really want to return null on this look up?

**Comment by jhulslander** [comment]
retest this please

File: src/test/java/edu/cornell/kfs/fp/batch/service/impl/AdvanceDepositServiceImplTest.java (hunk 1 above)
Comment (bhutchinson) [inline src/test/java/edu/cornell/kfs/fp/batch/service/impl/AdvanceDepositServiceImplTest.java]: I think it's fine. It's in a MockBusinessObjectService just used for a test class. This method was recently added, so I needed to implement it to fix the compiler error, but most of the methods in that Mock, included another findMatching version, return null.

Comment (jhulslander) [inline src/test/java/edu/cornell/kfs/fp/batch/service/impl/AdvanceDepositServiceImplTest.java]: oh, I see now, my bad

**Pull request merged**